src/client/AM.PA.MacOs/PersonalAnalytics/Trackers/UserInputTracker/Visualizations/ActivityVisualization.swift

```diff
@@ -18,11 +18,8 @@ class ActivityVisualization: IVisualization{
     //TODO
     var title: String
     var _type = [VisConstants.Day]
-    
-    let sql: InputSQLController
-    
+        
     required init() throws {
```

Comment:
not super important, but I dont think that the `throws` is necessary for any of the visualizations anymore? We should remove this

Reply:
You are right! removed it.